### PR TITLE
Fix getting deviceset name function

### PR DESCRIPTION
**Pull request by prsurve**
Fix for failure of test_disk_failures.py

Signed-off-by: prsurve <prsurve@redhat.com>

### CODE REVIEW

File: tests/manage/cluster/nodes/test_disk_failures.py [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
-from ocs_ci.framework.testlib import tier4, ignore_leftovers, ManageTest
+from ocs_ci.framework.testlib import tier4, ignore_leftovers, ManageTest, aws_platform_required
 from tests.sanity_helpers import Sanity
 from tests.helpers import wait_for_resource_count_change, get_admin_key
 
 logger = logging.getLogger(__name__)
 
 
 @tier4
+@aws_platform_required
```
Comment (ebenahar) [inline tests/manage/cluster/nodes/test_disk_failures.py]: I don't think this is needed. This test was adjusted to work with any platform

Comment (prsurve) [reply tests/manage/cluster/nodes/test_disk_failures.py]: I guess we dont have lib written for vmware [liburl](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/platform_nodes.py#L80-L129)

Comment (ebenahar) [reply tests/manage/cluster/nodes/test_disk_failures.py]: No, not yet

Comment (ebenahar) [reply tests/manage/cluster/nodes/test_disk_failures.py]: I mean, it's not implemented in https://github.com/red-hat-storage/ocs-ci/blob/1b1293d2b0c667fbe40ef8964c28897b20709b35/ocs_ci/ocs/platform_nodes.py#L80.
I'm planning to do that in the upcoming days :)

Comment (prsurve) [reply tests/manage/cluster/nodes/test_disk_failures.py]: thanks, I removed aws_platform_required marker

**Pull request merged**